examples/automm/object_detection/detection_train.py

```diff
@@ -66,35 +77,32 @@ def detection_train(
     start = time.time()
     predictor.fit(
         train_path,
+        tuning_data=val_path,
         hyperparameters={
-            "optimization.learning_rate": lr,
-            "optimization.weight_decay": wd,
+            "optimization.learning_rate": lr/100, # we use two stage and lr_mult=100 for detection
```

Comment:
Currently only last few layers will use full lr, i.e. `bbox_head.convs_pred` for `yolov3`, `roi_head.bbox_head.fc_cls` and `roi_head.bbox_head.fc_reg` for `faster-rcnn`. Because those layer are initialized from scratch after `num_classes` changed.

Reply:
I see, thanks.